exchanges/gdax.js

```diff
@@ -307,10 +307,20 @@ Trader.prototype.getMaxDecimalsNumber = function (number, decimalLimit = 8) {
   return decimalCount <= decimalLimit ? decimalNumber.toString() : (Math.floor(decimalNumber * decimalMultiplier) / decimalMultiplier).toFixed(decimalLimit);
 };
 
-Trader.prototype.deductFee = function (number, fee = 0.003) {
-    var resultNumber = parseFloat(number) * (1 - fee);
+Trader.prototype.deductFee = function (number) {
+    var _this = this;
```

Comment:
nitpick: you don't need to do this if you are creating a function using the fat arrow, that wil automatically bind context.